src-electron/sdk/matter.js

```diff
@@ -52,15 +37,11 @@ async function getForcedExternalStorageList(db, zcl) {
  * @returns An array of objects
  */
 
-async function getForcedExternalStorage(db, attributeId) {
-  let pkgs = await queryPackage.getPackageRefByAttributeId(db, attributeId)
-  let zcl = await queryPackage.getPackageByPackageId(db, pkgs)
-  zcl = zcl?.path
-  let obj = await fsp.readFile(zcl, 'utf-8')
-  let data = parseJson(obj)
-  let byName = data?.attributeAccessInterfaceAttributes
-  let lists = data?.listsUseAttributeAccessInterface
-  let forcedExternal = { byName, lists }
+async function getForcedExternalStorage(db) {
+  let forcedExternal = await queryPackage.getAttributeAccessInterface(
+    db,
+    '999999'
```

Comment:
Where does 999999 come from? same as above

Reply:
test code, needs to be cleaned